tests/test_serialization.py

```diff
@@ -0,0 +1,105 @@
+import gymnasium as gym
+import numpy as np
+import pytest
+
+from minari.serialization import deserialize_space, serialize_space
+
+
+@pytest.mark.parametrize(
+    "space",
+    [
+        gym.spaces.Box(low=-1, high=4, shape=(2,), dtype=np.float32),
```

Comment:
Can you delegate to test_common?

Reply:
Oh sounds good, I'll do it for test_serialization as well.

Reply:
done